pkg/cloudprovider/plugins/azure/azure_compute.go

```diff
@@ -40,16 +40,32 @@ type computeServiceConfig struct {
 	resourcesCache         *internal.CloudServiceResourcesCache
 	inventoryStats         *internal.CloudServiceStats
 	credentials            *azureAccountConfig
-	computeFilters         map[string][]*string
+	selectorConfigMap      map[types.NamespacedName]*selectorConfig
+	configChangedSelectors []*types.NamespacedName
+}
+
+// selectorConfig is meant for storing details about CES configuration, created query string etc.
+type selectorConfig struct {
+	namespacedName  types.NamespacedName
+	instanceFilters []*string
 	// selectors required for updating resource filters on account config update.
-	selectors map[string]*crdv1alpha1.CloudEntitySelector
+	ces *crdv1alpha1.CloudEntitySelector
 }
 
-type computeResourcesCacheSnapshot struct {
+type selectorLevelCacheSnapshot struct {
 	virtualMachines map[internal.InstanceID]*virtualMachineTable
-	vnets           []armnetwork.VirtualNetwork
-	vnetIDs         map[string]struct{}
-	vnetPeers       map[string][][]string
+}
+
+type accountLevelCacheSnapshot struct {
```

Comment:
Remove accountLevelCache and move it to CacheSnapshot

Reply:
done